src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -204,6 +204,12 @@ public static void CopyToCovariantArray()
         [InlineData((4L * 1024L * 1024L * 1024L) + 256)]
         public static void CopyToLargeSizeTest(long bufferSize)
         {
+            // If this test is run in a 32-bit process, the large allocation will fail.
+            if (Unsafe.SizeOf<IntPtr>() != sizeof(long))
```

Comment:
> It also relies on an unclear relationship between the size of IntPtr and the size of Int64.

Is the relationship unclear? I thought IntPtr can only be one of 32-bit or 64-bit.